spec/rubocop/target_finder_spec.rb

```diff
@@ -78,6 +20,64 @@
     create_file('dir1/executable',  '#!/usr/bin/env ruby')
     create_empty_file('dir2/ruby3.rb')
     create_empty_file('.hidden/ruby4.rb')
+
+    stub_const('RUBY_EXTENSIONS', %w[.rb
```

Comment:
Same as in one of the above cases - a variable, `let` or a method would be preferrable. `stub_const` is unnecessary.

Reply:
:+1: Will fix.